akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -235,7 +235,10 @@ object Sink {
   /**
    * A `Sink` that will consume the stream and discard the elements.
    */
-  def ignore: Sink[Any, Future[Done]] = fromGraph(GraphStages.IgnoreSink)
+  def ignore: Sink[Any, Future[Done]] =
+    Flow[Any].log("ignoreSink").toMat(ignoreWithoutLogging)(Keep.right)
```

Comment:
As far as I checked, `def foreach` and `def onComplete` of the same `Sink` object (and Java API of `ignore`/`foreach`), which I already updated, were the only places calling this `ignore` other than the test code.